locale/en/site.json

```diff
@@ -72,16 +84,18 @@
             "link": "docs/faq",
             "text": "FAQ"
         },
-        "api": {
-            "text": "API"
+        "api-lts": {
+            "link": "dist/latest-v4.x/docs/api",
```

Comment:
Is this ok?
I think that  `dist/latest-v4.x/docs/api` becomes `/en/dist/latest-v4.x/docs/api/` after generation.
However true url is `https://nodejs.org/dist/latest-v4.x/docs/api/`.

`dist/latest-v5.x/docs/api` is also the same.